src/pages/Missions.js

```diff
@@ -1,6 +1,21 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getMissions } from '../redux/missions/missionSlice';
 
 function Missions() {
+  /* Setup Redux dispatch */
+  const dispatch = useDispatch();
+
+  /* At the beginning cont => [], after dispatch cont => [{},{}...{}] */
+  const cont = useSelector((state) => state.missions);
+
+  /* For dispatching only once */
+  useEffect(() => {
+    if (cont.length === 0) {
+      dispatch(getMissions());
+    }
+  }, [dispatch, cont]);
```

Comment:
Please remove the state from the useEffect dependencies so that it does not fetch data after every change to the state as required:
 #30 
> missions data should only be fetched (once) when a user navigates to the Missions section.